recipes/cminpack/all/conanfile.py

```diff
@@ -0,0 +1,105 @@
+from conan.tools.cmake import CMake, CMakeToolchain, CMakeDeps, cmake_layout
+from conan.tools import files
+from conan import ConanFile
+import os
+import textwrap
+
+required_conan_version = ">=1.45.0"
+
+
+class CMinpackConan(ConanFile):
+    name = "cminpack"
+    url = "https://github.com/conan-io/conan-center-index"
+    description = "About A C/C++ rewrite of the MINPACK software (originally in FORTRAN)" \
+                  "for solving nonlinear equations and nonlinear least squares problems"
+    topics = ("nonlinear", "solver")
+    homepage = "http://devernay.free.fr/hacks/cminpack/"
+    license = "BSD-like"
```

Comment:
It does not fit an official SPDX license, but there is no exception. So, BSD-like sounds okay here.

Reply:
Yeah... I basically quoted the text from the homepage 

> Minpack is licensed under a BSD-like license (available in the distribution). .... It [Cminpack]is distributed under the original Minpack license (see the file CopyrightMINPACK.txt in the distribution).

Reply:
Please follow https://github.com/conan-io/conan-center-index/blob/master/docs/faqs.md#what-license-should-i-use-for-a-custom-project-specific-license 🙏

Reply:
```suggestion
    license = "LicenseRef-CopyrightMINPACK.txt"
```